card/mod/05_standard/set/type/signup.rb

```diff
@@ -96,18 +96,23 @@ def has_token?
 end
 
 event :activate_account do
-  subcards['+*account'] = {'+*status'=>'active'}
+  add_subfield :account
+  subfield(:account).add_subfield :status, content: 'active'
   self.type_id = Card.default_accounted_type_id
   account.send_welcome_email
 end
 
-event :approve_with_token, on: :update, before: :approve, when: proc {|c| Env.params[:approve_with_token] } do
+event :approve_with_token,
+      on: :update, before: :approve,
+      when: proc { |c| Env.params[:approve_with_token] } do
```

Comment:
Unused block argument - `c`. You can omit the argument if you don't care about it.